cmd/mount.go

```diff
@@ -248,13 +250,15 @@ var mountCmd = &cobra.Command{
 		}
 
 		if configFileExists {
-			parseConfig()
+			err := parseConfig()
+			if err != nil {
+				return err
```

Comment:
We should keep this consistent in all the places where parseConfig() method is called